catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -196,19 +197,22 @@ public UpdateResponse update(UpdateStorageRequest streamUpdateRequest)
                 updateStorageResponse = processPostUpdateStoragePlugins(updateStorageResponse);
 
                 for (ContentItem contentItem : updateStorageResponse.getUpdatedContentItems()) {
-                    Metacard metacard = metacardMap.get(contentItem.getId());
+                    if (StringUtils.isBlank(contentItem.getQualifier())) {
```

Comment:
This now matches [the corresponding method in `CreateOperations`](https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java#L656).